lib/bundle.js

```diff
@@ -103,31 +199,33 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     return systemBuilder.trace(firstModule, cfg);
```

Comment:
Here first module needs to be a no glob operation at the moment as `firstmodule` is being treated specially for generating the initial `trace`. We have to start the loop here instead as we did in the next `then` to support `glob` in any position.